### ISSUE TITLE
Object Required Error in Microsoft Word

### ISSUE THREAD

**Issue by klwjan** [xlwings/xlwings#1045]
In the latest version of xlwings (v0.15.3), although the UDF is working fine in Microsoft Excel, exporting UDFs to Microsoft Word results in a Object Required Error 424. 

The same set of codes/UDF exports is working properly for v0.15.1.

**Comment by fzumstein** [comment]
First of all, it's great to hear that this feature is used at all!

I just tried it without issues. Are you sure you changed the first line in both the `xlwings` and the `xlwings_udfs` modules to: `#Const App = "Microsoft Word"`?

**Comment by klwjan** [comment]
> 
> 
> First of all, it's great to hear that this feature is used at all!

It's great that this function existed :)

>
> I just tried it without issues. Are you sure you changed the first line in both the `xlwings` and the `xlwings_udfs` modules to: `#Const App = "Microsoft Word"`?

Yes, both modules have been changed. Trust Center settings have also been changed to enable VBA macro. It is strange as the setup for v0.15.1 was exactly the same and it worked.

**Comment by fzumstein** [comment]
Are you able to upload a dummy file so I can replicate?

**Comment by klwjan** [comment]
> 
> 
> Are you able to upload a dummy file so I can replicate?

I just did some testing again, it works when it is attached to the current document, but I want to attach it to 'Normal'. You might be able to replicate the error if you place the UDF in 'Normal' instead of the current document/Project. 
I've attached the dummy files just in case it would be better to have it.
[dummy.zip](https://github.com/ZoomerAnalytics/xlwings/files/2913156/dummy.zip)
